chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/MergeDeveloperJob.java

```diff
@@ -156,6 +158,13 @@ private void setSecurityContext(UserDTO user) {
         SecurityContextHolder.setStrategyName(SecurityContextHolder.MODE_INHERITABLETHREADLOCAL);
     }
 
+    private void checkPreMergeDevelopers() throws EntityRetrievalException {
```

Comment:
This method looks like all it does is read Developers. I'm guessing the point of the method is to throw an exception if there's something odd that happens. If that's true, it feels like the method should be named differently, or called differently, or something, so that someone looking at this code in 6 months will recognize that the point is that we're checking to see if an "EntityRetrievalException" is thrown or not